python/tank/util/qt_importer.py

```diff
@@ -207,6 +246,9 @@ def _import_pyqt4(self):
         QtCore.Slot = QtCore.pyqtSlot
         QtCore.Property = QtCore.pyqtProperty
 
+        QtNetwork = self._import_module_by_name("PyQt4", "QtNetwork")
```

Comment:
The helper is there specifically to avoid this four line pattern that would repeat all over the methods.